docs/reports/v0.2.0/evaluation.log

```diff
@@ -0,0 +1,1793 @@
+github.com/symflower/eval-codegen-testing/cmd/eval-codegen-testing/cmd.(*Evaluate).Execute
+        /home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/cmd/eval-codegen-testing/cmd/evaluate.go:115
+github.com/jessevdk/go-flags.(*Parser).ParseArgs
+        /home/zimmski/symflower/pkg/mod/github.com/jessevdk/go-flags@v1.5.1-0.20210607101731-3927b71304df/parser.go:335
+github.com/jessevdk/go-flags.(*Parser).Parse
+        /home/zimmski/symflower/pkg/mod/github.com/jessevdk/go-flags@v1.5.1-0.20210607101731-3927b71304df/parser.go:191
+github.com/symflower/eval-codegen-testing/cmd/eval-codegen-testing/cmd.Execute
+        /home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/cmd/eval-codegen-testing/cmd/command.go:26
+main.main
+        /home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/cmd/eval-codegen-testing/main.go:8
+runtime.main
+        /home/zimmski/symflower/.devenv/go@1.21.5/go/src/runtime/proc.go:267
+runtime.goexit
+        /home/zimmski/symflower/.devenv/go@1.21.5/go/src/runtime/asm_amd64.s:1650
+plain.go]
+2024/04/03 23:07:13 Evaluating model "openrouter/xwin-lm/xwin-lm-70b" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain"
+2024/04/03 23:07:14 Evaluated model "openrouter/xwin-lm/xwin-lm-70b" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain": encountered 1 problems
+2024/04/03 23:07:14 Excluding model "openrouter/xwin-lm/xwin-lm-70b" since it was not able to solve the "plain" repository for language "golang": [empty LLM "xwin-lm/xwin-lm-70b" response: {ID: Object: Created:0 Model: Choices:[] Usage:{PromptTokens:0 CompletionTokens:0 TotalTokens:0} SystemFingerprint: httpHeader:map[Access-Control-Allow-Credentials:[true] Access-Control-Allow-Headers:[Authorization, User-Agent, X-Api-Key, X-CSRF-Token, X-Requested-With, Accept, Accept-Version, Content-Length, Content-MD5, Content-Type, Date, X-Api-Version, HTTP-Referer, X-Windowai-Title, X-Openrouter-Title, X-Title, X-Stainless-Lang, X-Stainless-Package-Version, X-Stainless-OS, X-Stainless-Arch, X-Stainless-Runtime, X-Stainless-Runtime-Version] Access-Control-Allow-Methods:[GET,OPTIONS,PATCH,DELETE,POST,PUT] Access-Control-Allow-Origin:[*] Cache-Control:[public, max-age=0, must-revalidate] Cf-Cache-Status:[DYNAMIC] Cf-Ray:[86ec06eb8928722d-VIE] Content-Type:[application/json] Date:[Wed, 03 Apr 2024 21:07:14 GMT] Server:[cloudflare] Strict-Transport-Security:[max-age=63072000] X-Matched-Path:[/api/v1/chat/completions] X-Vercel-Id:[fra1::9cr65-1712178433866-8ab417015282]]}
+plain.go]
+2024/04/03 23:07:14 Evaluating model "symflower/symbolic-execution" using language "golang" and repository "/home/zimmski/symflower/src/github.com/symflower/eval-symflower-codegen-testing/testdata/golang/plain"
```

Comment:
@Munsio can you also add to your task list to change the repository path to be relative not absolute in the logs. The information should be as stable as possible for everyone. No matter where you run the evalation.